### PR TITLE
added course_is_public_to_auth_users to create course call, updated s…

### PR DESCRIPTION
**Pull request by eparker71**
…tatic methods

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,3 +68,5 @@ docs/_build/
 
 # PyCharm IDE metadata
 .idea/
+
+.DS_Store
```
Comment (elliottyates) [inline .gitignore]: (.DS_Store should be in a personal .gitignore, not in the project .gitignore)

File: canvas_sdk/methods/courses.py [hunk 2]
```diff
@@ -63,6 +63,8 @@ def create_new_course(request_ctx, account_id, course_name=None, course_course_c
         :type course_license: string or None
         :param course_is_public: (optional) Set to true if course if public.
         :type course_is_public: boolean or None
+        :param course_is_public_to_auth_users: (optional) Set to true if course if public to autorized users.
```
Comment (elliottyates) [inline canvas_sdk/methods/courses.py]: typo: *is* public to *authorized*

File: canvas_sdk/methods/courses.py [hunk 3]
```diff
@@ -592,6 +595,8 @@ def update_course(request_ctx, id, course_account_id=None, course_name=None, cou
...
```
Comment (elliottyates) [inline canvas_sdk/methods/courses.py]: same typo

File: static_methods/courses.py [hunk 4]
```diff
@@ -6,6 +6,96 @@
...
+        :type course_course_code: string or None
+        :param course_start_at: (optional) Course start date in ISO8601 format, e.g. 2011-01-01T01:00Z
+        :type course_start_at: datetime or None
+        :param course_end_at: (optional) Course end date in ISO8601 format. e.g. 2011-01-01T01:00Z
+        :type course_end_at: datetime or None
+        :param course_license: (optional) The name of the licensing. Should be one of the following abbreviations (a descriptive name is included in parenthesis for reference): - 'private' (Private Copyrighted) - 'cc_by_nc_nd' (CC Attribution Non-Commercial No Derivatives) - 'cc_by_nc_sa' (CC Attribution Non-Commercial Share Alike) - 'cc_by_nc' (CC Attribution Non-Commercial) - 'cc_by_nd' (CC Attribution No Derivatives) - 'cc_by_sa' (CC Attribution Share Alike) - 'cc_by' (CC Attribution) - 'public_domain' (Public Domain).
+        :type course_license: string or None
+        :param course_is_public: (optional) Set to true if course if public.
+        :type course_is_public: boolean or None
+        :param course_is_public_to_auth_users: (optional) Set to true if course if public to autorized users.
```
Comment (elliottyates) [inline static_methods/courses.py]: Same typos :)

**Comment by elliottyates** [comment]
:+1: with a few minor typo changes and removing the .gitignore change

**Comment by elliottyates** [comment]
Also may want to consider bumping version in another commit before merging.

**Comment by eparker71** [comment]
re version, sure, I didn't know if anyone else had an update for this as part of another story. I wanted to double check before merging.

**Comment by eparker71** [comment]
as far as gitignore, I don't' recall seeing DS_store files before. Was that removed?

**Pull request merged**